pages/blog/articles/[id].tsx

```diff
@@ -55,7 +57,8 @@ export async function getStaticPaths() {
 }
 
 export const getStaticProps: GetStaticProps<BlogPostProps> = async ({ params }) => {
-  const article = await getArticleBySlug(params.slug as string)
+  console.log('params', params)
+  const article = await getArticleBySlug(params.id as string)
```

Comment:
Shouldn't params props be typed?

Reply:
Okay, will make a private function for it

Reply:
This is how it was done in all the app before :) 
Not meaning that it was good. I've improved it with this commit. cc/ @fairlighteth  https://github.com/cowprotocol/cow-fi/pull/193/commits/5e4e7a3414caf3b5240d44e2dec55cfab056c14a

Reply:
Note that we are also more specific with the param name now: 
<img width="921" alt="image" src="https://github.com/cowprotocol/cow-fi/assets/2352112/f4a1c968-2d1b-4bf5-8a4d-8227f8ffe2a6">